### ISSUE TITLE
[a11y]: - Radio Button - Playground - JAWS is not announcing the read only property

### ISSUE THREAD

**Issue by guidari** [carbon-design-system/carbon#16607]
### Package

@carbon/react

### Browser

Firefox

### Package version

v1.58.0

### React version

18.02

### Automated testing tool and ruleset

Manual QA

### Assistive technology

JAWS

### Description

From the playground set the read only option to "True", Observe the JAWS announcement. JAWS is not announcing the radio buttons as "read only".

### Recommend Fix
If the radio buttons is read-only then jaws should announce as the radio button as read-only.

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-radiobutton--playground&args=readOnly:!true

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Since this playground example doesn’t exist in our Storybook, we’re not able to reproduce or verify the issue as described. I believe we can close this issue.
